packages/desktop-client/src/components/payees/ManagePayeesWithData.js

```diff
@@ -63,7 +63,7 @@ export default function ManagePayeesWithData({ initialSelectedIds }) {
     };
   }, []);
 
-  async function onUndo({ tables, messages, meta, url }, scroll = false) {
+  async function onUndo({ tables, messages, meta }, _scroll = false) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏can just remove it if it's not used

```suggestion
  async function onUndo({ tables, messages, meta }) {
```